edith-step/src/index.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
 
```

Comment:
Lint errors:

   9:1   error  Expected indentation of 4 spaces but found 2  indent
  10:1   error  Expected indentation of 6 spaces but found 4  indent
  11:1   error  Expected indentation of 4 spaces but found 2  indent
  12:1   error  Expected indentation of 4 spaces but found 2  indent
  12:34  error  Missing trailing comma                        comma-dangle

Please fix.